### PR TITLE
add two more lock options

### PR DESCRIPTION
**Pull request by oxSaturn**
Closes https://app.dework.xyz/velocimeter/main-space-58499?taskId=7764d010-a3a6-4f03-925d-3ebccabcde21

### CODE REVIEW

File: Frontend-v1-Original/components/ssVest/lock.tsx [hunk 1]
```diff
@@ -332,39 +318,22 @@ export default function ssLock({
...
-              />
+              {Object.keys(lockOptions).map((key) => {
+                return (
+                  <FormControlLabel
+                    key={key}
+                    className={classes.vestPeriodLabel}
+                    value={lockOptions[key]}
+                    control={<Radio color="primary" />}
+                    label={key}
+                    labelPlacement="end"
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/ssVest/lock.tsx:333]: I've changed `labelPlacement` from `start` to `end` as I found the `start` one counter-intuitive and hard to read.

File: Frontend-v1-Original/components/ssVest/ssVest.module.css [hunk 2]
```diff
@@ -94,6 +94,7 @@
   justify-content: flex-end;
   position: absolute;
   top: 8px;
+  z-index: 100;
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/ssVest/ssVest.module.css:97]: You can't click the `balance` to fill the text field as it's hidden by other layer with a `z-index` of 99.

File: Frontend-v1-Original/components/ssVest/lock.tsx [hunk 3]
```diff
@@ -332,39 +318,22 @@ export default function ssLock({
           <div className={classes.inline}>
             <Typography className={classes.expiresIn}>Expires: </Typography>
             <RadioGroup
-              className={classes.vestPeriodToggle}
-              row
+              className={`${classes.vestPeriodToggle} grid grid-cols-2`}
```
Comment (t0rbik) [inline Frontend-v1-Original/components/ssVest/lock.tsx:321]: thanks for using tailwind, idea is to switch from css modules completely as i find css modules:
1. lots of dublicated code
2. expensive to maintain
3. bad dx

File: Frontend-v1-Original/components/ssVest/lockDuration.tsx [hunk 4]
```diff
@@ -172,39 +167,22 @@ export default function ffLockDuration({
...
+                <FormControlLabel
+                  key={key}
+                  className={classes.vestPeriodLabel}
+                  value={lockOptions[key]}
+                  control={<Radio color="primary" />}
+                  label={key}
+                  labelPlacement="end"
+                />
+              );
+            })}
```
Comment (t0rbik) [inline Frontend-v1-Original/components/ssVest/lockDuration.tsx:185]: 🔥

File: Frontend-v1-Original/components/ssVest/ssVest.module.css (hunk 2 above)
Comment (t0rbik) [reply Frontend-v1-Original/components/ssVest/ssVest.module.css:97]: sounds good

**Review by t0rbik — APPROVED** [review]
let me authorise vercel and see if all good

File: Frontend-v1-Original/components/ssVest/lock.tsx (hunk 3 above)
Comment (oxSaturn) [reply Frontend-v1-Original/components/ssVest/lock.tsx:321]: yup, I've been using tailwind for all my works when possible. The best css solution at the moment.

**Pull request merged**